repo/packages/P/prometheus/0/marathon.json

```diff
@@ -0,0 +1,47 @@
+{
+  "id": "prometheus",
+  "cpus": 1,
+  "mem": 2048,
+  "requirePorts": true,
+  "instances": 1,
```

Comment:
cpus, mem and instances have to be configurable with defaults defined in config.json.

id must be configurable as well, and should use the prometheus .framework-name property (which is missing in your config)